src/libraries/System.Private.Xml/src/System/Xml/Base64Decoder.cs

```diff
@@ -184,7 +171,7 @@ private static byte[] ConstructMapBase64()
                 int digit;
                 if (ch > 122 || (digit = s_mapBase64[ch]) == Invalid)
                 {
-                    throw new XmlException(SR.Xml_InvalidBase64Value, new string(pChars, 0, (int)(pCharsEndPos - pChars)));
+                    throw new XmlException(SR.Xml_InvalidBase64Value, chars.ToString());
```

Comment:
I might be missing somehting but not seeing you slicing `chars` anywhere. Should this be something along the lines of `chars.Slice(0, char.Length - iChar)`?

Reply:
ahh, it's `pChars` not `pChar`